frontend/src/__tests__/unit/utilities/addTypesToK8sListedReources.spec.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  mockTemplateK8sResource,
+  mockTemplateK8sResourceWithoutKind,
+} from '~/__mocks__/mockServingRuntimeTemplateK8sResource';
+
+test('should have kind as Template', () => {
+  const servingRuntimeTemplate = mockTemplateK8sResource({});
+  expect(servingRuntimeTemplate.kind).toBe('Template');
+});
+test('should not have kind', () => {
+  const servingRuntimeTemplateWithoutKind = mockTemplateK8sResourceWithoutKind({});
+  expect(servingRuntimeTemplateWithoutKind).not.toContain('kind');
+});
```

Comment:
You can now co-locate your `__tests__` dir near the target source file.

Also, please add tests for `addTypesToK8sListedResources`.